src/interface/rgb21.rs

```diff
@@ -138,6 +139,21 @@ pub struct EmbeddedMedia {
     pub data: SmallBlob,
 }
 
+impl EmbeddedMedia {
+    pub fn from_strict_val_unchecked(value: &StrictVal) -> Self {
+        let ty = MediaType::from_strict_val_unchecked(value.unwrap_struct("type"));
+        let data = SmallBlob::from_collection_unsafe(
+            value
+                .unwrap_struct("data")
+                .unwrap_bytes()
+                .try_into()
+                .expect("invalid data"),
+        );
```

Comment:
Fixes linter warning:

```
use of a fallible conversion when an infallible one could be used
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_fallible_conversions
```

```suggestion
        let data =
            SmallBlob::from_collection_unsafe(value.unwrap_struct("data").unwrap_bytes().into());
```